src/main/app/case/C100CaseApi.ts

```diff
@@ -106,10 +148,8 @@ export class CaseApi {
       if (!caseId) {
         throw new Error('caseId not found so case could not be deleted.');
       }
-      await this.axios.post<UpdateCaseResponse>(`${caseId}/${C100_CASE_EVENT.DELETE_CASE}/update-case`, caseData, {
-        headers: {
-          accessCode: 'null',
-        },
+      await this.axios.post<UpdateCaseResponse>(`/citizen/${caseId}/delete-application`, caseData, {
+        headers: {},
```

Comment:
Do we not need auth headers?

Reply:
Its kind of common. that client itself contains this details.